### PR TITLE
[TOOLS-4521] update CMT license

### PR DESCRIPTION
**Pull request by Kang-dot**
http://jira.cubrid.org/browse/TOOLS-4521

Purpose
update CMT license strings and change splash image

Implementation
N/A

Remarks
N/A

### CODE REVIEW

**Review by kisoo-han — APPROVED** [review]

File: com.cubrid.common.update.feature/feature.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 featureName=CUBRID Tools Common Update Features
 providerName=Search Solution Corporation
 description=CUBRID Tools Common Update Feature
-copyright=Copyright (C) 2014 Search Solution Corporation. All rights reserved by Search Solution.
+copyright=Copyright (C) 2016 CUBRID Corporation. All rights reserved by Search Solution.
```
Comment (Srltas) [inline com.cubrid.common.update.feature/feature.properties]: Shouldn't it be changed to `All rights reserved by CUBRID.`? or how about changing the whole sentence to `Copyright (C) 2016 CUBRID Corporation all rights reserved.`?

File: com.cubrid.common.update.feature/feature.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 licence=BSD Licence\n\
 \n\
 Copyright (C) 2014 Search Solution Corporation. All rights reserved by Search Solution.\n\
```
Comment (Srltas) [inline com.cubrid.common.update.feature/feature.properties]: Is there a reason why you didn't change this part?

File: com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/message/Messages.properties [hunk 3]
```diff
@@ -1,4 +1,4 @@
-aboutMessage={0} {1} build {2}\r\n\r\nCopyright (C) 2014 Search Solution Corporation.\r\nAll rights reserved by Search Solution.\r\n\r\nHome page\: {3}\r\nCUBRID Database project\: {4}\r\nCUBRID Tools project\: {5}
+aboutMessage={0} {1} build {2}\r\n\r\nCopyright (C) 2016 CUBRID Corporation.\r\nAll rights reserved by Search Solution.\r\n\r\nHome page\: {3}\r\nCUBRID Database project\: {4}\r\nCUBRID Tools project\: {5}
```
Comment (Srltas) [inline com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/message/Messages.properties]: This part also seems to need to be revised.

**Review by hwany7seo — APPROVED** [review]

File: com.cubrid.common.update.feature/feature.properties (hunk 1 above)
Comment (Kang-dot) [reply com.cubrid.common.update.feature/feature.properties]: fix missing text in next commit. thanks

File: com.cubrid.common.update.feature/feature.properties (hunk 2 above)
Comment (Kang-dot) [reply com.cubrid.common.update.feature/feature.properties]: same as above

File: com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/message/Messages.properties (hunk 3 above)
Comment (Kang-dot) [reply com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/message/Messages.properties]: same as above

**Review by Srltas — APPROVED** [review]

**Pull request merged**